tests/test_api.js

```diff
@@ -0,0 +1,199 @@
+import request from "supertest";
+import { expect } from "chai";
+import app from "../app/app.js";
+import db from "../app/db.js";
+
+describe("Gamer API", () => {
+  // Clean up the database before each test
+  beforeEach(async () => {
+    await db.query("DROP TABLE IF EXISTS users_test.gamer_profile_test");
+    await db.query(
+      "CREATE TABLE users_test.gamer_profile_test (LIKE users_test.gamer_profile INCLUDING ALL)",
+    );
+  });
+
```

Comment:
I'm unsure how mocha worths with express. Are you creating mocks for all the tests here or are you actually creating/edititing rows in the table?

Reply:
Creates/edits rows to a test table  that lives in our database, not the live table where we'll have our real profiles

Reply:
Oh I see I read this wrong. So you create a table if it doens't exit and if it does it drops it? I'm good with that. I thought we were just creating a bunch of users/tables without ever cleaning it up.